chance.js

```diff
@@ -2294,6 +2294,28 @@ options,
         return this.pick(this.currency_types());
     };
 
+    // return random world amount of currency
+    Chance.prototype.amount = function (options) {
+        options = initOptions(options, { min: 0, max: 10000, fixed: 2 });
```

Comment:
Also please allow specifying a currencyCode as an option in case the developer wants a random amount value but not a random currencyCode

Reply:
added custom `currency` options to the amount